src/test/kotlin/kitchenops/application/MenuGroupServiceTest.kt

```diff
@@ -0,0 +1,73 @@
+package kitchenops.application
+
+import io.kotest.matchers.shouldBe
+import io.mockk.every
+import io.mockk.mockk
+import io.mockk.verify
+import kitchenpos.application.MenuGroupService
+import kitchenpos.domain.MenuGroup
+import kitchenpos.domain.MenuGroupRepository
+import spec.BaseUnitSpec
+import java.util.*
+
+internal class MenuGroupServiceTest : BaseUnitSpec({
+
+    val menuGroupRepository = mockk<MenuGroupRepository>()
+    val sut = MenuGroupService(menuGroupRepository)
+
+    context("메뉴 그룹을 생성할 수 있다") {
+        test("이름을 지정하지 않으면 생성할 수 없다") {
+            // given
+            val request = MenuGroup().apply { name = null }
+
+            // when
+            val actual = runCatching { sut.create(request) }
+
+            // then
+            actual.exceptionOrNull() shouldBe IllegalArgumentException()
```

Comment:
kotest에서 제공해주는 검증문을 활용해보면 어떨까요? 😃 

```suggestion
            // when // then
            shouldThrow<IllegalArgumentException> { sut.create(request) }
```

개인적으로 `runCatching`을 잘 사용해보지는 않아서, 검증문으로 쓰는게 크게 문제되보이지는 않지만, 가급적 검증 용도로 제공되는 함수를 쓰는게 더 좋다고 생각해요. 🤔

Reply:
테스트를 작성할 때 given, when, then 순서로 작성하는데 exception 검증은 when then의 순서가 거꾸로 되어있어서 개인적으로는 코드 형태가 일관적이지 않다고 생각합니다.
shouldThrow 내부도 block을 호출하고 에러가 나오는지 try catch로 잡아서 동작하는 형식이라 의미가 동일하고, 테스트 에러 메시지도 동일하게 나와서 이런 식으로 작성하곤 합니다.

Reply:
given, when, then의 주석 포맷에서 대략적으로 느끼고있었지만 테스트를 BDD 스타일로 잘 작성하고 계시군요? 👍💯